src/github3/apps.py

```diff
@@ -154,29 +154,45 @@ def _update_attributes(self, json):
         self.updated_at = self._strptime(json["updated_at"])
 
 
-def create_token(private_key_pem, app_id, expire_in=TEN_MINUTES_AS_SECONDS):
+def create_token(
+    private_key_pem,
+    app_id,
+    expire_in=TEN_MINUTES_AS_SECONDS,
+    token_creator=None,
```

Comment:
Thanks for this PR. I like this change, but adding a function parameter could be complicated for people wanting to use that without type hints. As a suggestion, I think it would be great if you could annotate the parameters of create_token().

Reply:
That's a good point, I have added a bit to the sphinx documentation and used the typing module for the whole signature.